### PR TITLE
Chore: rm .html files on S3

### LINKED ISSUE #1029: Deployment job: don't upload unused .html files.
The current deployment script uploads everything in the build output directory, and then additionally uploads HTML files w/o an extension.
Ideally, we should not upload the `.html` files, it will save some space.

image (1)

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

Resolves #1029

## How this PR fixes it

Excludes HTML files when syncing.

### CODE REVIEW

File: scripts/github/s3_upload.sh [hunk 1]
```diff
@@ -4,16 +4,19 @@ set -ev
...
 
-# Finally, upload all HTML files again but w/o an extention so that URLs like /welcome open the right page
+# Upload HTML files but w/o an extention so that URLs like /welcome open the right file
 for file in $(find . -name '*.html' | sed 's|^\./||'); do
-    aws s3 cp ${file%} $BUCKET/${file%.*} --content-type 'text/html'
+    aws s3 cp ${file%} $BUCKET/${file%.*} --content-type "text/html"
 done
 
+# Upload index.html as the default file
+aws s3 cp index.html $BUCKET
```
Comment (katspaugh) [inline scripts/github/s3_upload.sh:20]: WFYT @luarx? This avoids uploading unused .html files, but we then have to upload `index.html` explicitly. Do you think it's worth it?

**Comment by katspaugh** [comment]
Closing because the complexity overhead in the upload script is not worth the extra few KBs.

**Comment by luarx** [comment]
Ok

File: scripts/github/s3_upload.sh (hunk 1 above)
Comment (luarx) [reply scripts/github/s3_upload.sh:20]: Maybe not

**Pull request closed without merging**